docs/modules/docker_compose.md

```diff
@@ -119,6 +119,18 @@ public static DockerComposeContainer environment =
             .waitingFor("db_1", Wait.forLogMessage("started", 1))
             .withLocalCompose(true);
 ```
+
+## Compose V2
+
+[Compose V2 is GA](https://www.docker.com/blog/announcing-compose-v2-general-availability/) and it relies on the `docker` command itself instead of `docker-compose`.
+
+```java
+public static DockerComposeContainer environment =
+    new DockerComposeContainer(new File("src/test/resources/compose-test.yml"))
+            .withExposedService("redis-1", REDIS_PORT, Wait.forListeningPort())
+            .waitingFor("db-1", Wait.forLogMessage("started", 1));
+```
+
```

Comment:
The difference in the code example is the format of the services, does this mean it will be a breaking change for users to go with v2? Just realized this now while looking at the example.

Reply:
PR description has been updated taking into account all the information needed in this change.